okhttp/src/main/java/okhttp3/HttpUrl.kt

```diff
@@ -753,6 +753,9 @@ class HttpUrl internal constructor(builder: Builder) {
    */
   fun resolve(link: String): HttpUrl? = newBuilder(link)?.build()
 
+  /**
+   * Returns a builder based on this URL
```

Comment:
https://publicobject.com/2007/08/05/stop_writing_redundant_javadoc/

Reply:
I felt this was necessary because while it seemed reasonable that this started with HttpUrl's values, it also seemed a possibility that it started "blank" - hence why it seemed like it might be worthwhile to add a small bit of doc clarifying this.
If you'd rather see it removed, or additional information added to make it a little less basic, I can alter the change

Reply:
Sounds good. Add a period on the end of the sentence please.

Reply:
Period added, sorry for missing that convention from the other doc :)